release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -168,13 +168,22 @@ If you are upgrading your cluster to {product-title} 4.6, old tokens from {produ
 [id="ocp-4-6-machine-api"]
 === Machine API
 
+[id="ocp-4-6-machine-api-multiple-block-device-mappings"]
+==== Support for multiple block device mappings
+
+The Machine API now supports multiple block device mappings. Logs, data in empty directory pods, and docker images are now stored in separate block devices from the root device on a machine.
+
+[id="ocp-4-6-default-validation-providerspec-apis"]
+==== Defaults and validation for the Machine API `providerSpec`
+
+Defaults and validation are now enabled on a particular cloud provider API before input from the `providerSpec` is persisted to etcd. The Machine API Operator (MAO) now exposes both a `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` to the Cluster Version Operator (CVO), which run over the machine resource so that each provider can define their API defaults and validation.
```

Comment:
This explains what the action has been, but isn't really explaining the user impact.

The impact to a user is that validation is run against Machines/MachineSets at the time they are created to give feedback to the user in cases where the configuration is known to prevent machines from being created on the cloud provider side. For example if the user has provided no location information, the MachineSet will be rejected as this is a required parameter